interview-questions.md

```diff
@@ -50,7 +51,7 @@ action: "GET"    location: /students/2/edit
 action: "PATCH"  location: /students/2      
 - update the 2nd student
 action: "DELETE" location: /students/2      
-- delete the 2nd student
+- delete the 2nd student.
 
 
 5. As a developer, you are making an application to manage your to do list. Create 10 user stories that will help you get your application started. Read more about [user stories](https://www.atlassian.com/agile/project-management/user-stories).
```

Comment:
This is almost correct. What we want to describe is what the user will experience on our platform so we can build out the features that will match that experience. 

"As a user I can see a plus button that will navigate me to a page where I can create a to-do item"